src/Database/index.js

```diff
@@ -126,6 +134,23 @@ export default class Database {
     return merge$(...changesSignals).pipe(startWith(null))
   }
 
+  _subscribers: Array<[TableName<any>[], () => void]> = []
+
+  // Notifies `subscriber` on change in any of passed tables (only a signal, no change set)
```

Comment:
Because it was more work for me, and for the computers, and currently we have no use case where we actually need to subscribe to actual changes for multiple tables, just the signal that we need to re-query the database. We can always add it later without making backwards-incompatible changes.